### PR TITLE
Update ITEM_en.tsv

### PR DESCRIPTION
**Pull request by Sunaries**
Minor update.

### CODE REVIEW

File: ITEM_en.tsv [hunk 1]
```diff
@@ -2458,7 +2458,7 @@ ITEM_20150317_002457	Pokuborn Meat
 ITEM_20150317_002458	
 ITEM_20150317_002459	Loxodon Meat
 ITEM_20150317_002460	
-ITEM_20150317_002461	Ansr
+ITEM_20150317_002461	Ansur
```
Comment (dlam) [inline ITEM_en.tsv:2461]: 안스르 -> Ansr

From the [wiki](https://github.com/Treeofsavior/EnglishTranslation/wiki/2.4.%20Item)

**Comment by Sunaries** [comment]
The Wiki isn't always right since it was updated a few times. Ansur sounds better than Ansr.

**Comment by dlam** [comment]
I agree that the wiki isn't always right, but it's important to be consistent... should probably open an issue to update the wiki with the preferred romanization of 안스르.

**Comment by imcgames** [comment]
Updated "Anusr" in Wiki

**Pull request merged**